docs/kit/logistics/kit-loan-extension.md

```diff
@@ -0,0 +1,27 @@
+# Kit Loan Extension
+
+As per appendix A.2 of the [rules][rules], teams may request to keep the kit after the competition.
+
+## Requesting a loan extension
+
+Instructions for teams to request a loan extension are located in A.2 of the [rules][rules]. Generally this involves emailing Student Robotics before a specific deadline. Requests which come in after this deadline should be denied, unless there are exceptional circumstances.
+
+The rules do not define how long the loan extension can be given for. Historical this has been anywhere to a few weeks after the competition to 2 months. This is the date the kit should be returned, rather than when the teams dispatch it.
+
+If a team is granted a loan extension, make sure they are aware, and explicitly note when the kit is expected back.
+
+[Example email for kit loan extension](https://github.com/srobo/team-emails/blob/master/SR2019/2019-03-11-competition-information.md#kit-loan-extension)
+
+## Kit Return Pack
+
+Teams who have requested loan extensions should be issued with kit return packs at the end of the competition. The pack is mostly comprised of jiffy bags suitable for the fragile boards, and stickers. Teams should be instructed to securely pack and plastic wrap the RUBs for shipping, ensuring the contents can't roll around.
+
+## Returning the kit
+
+The teams are responsible for arranging and paying for a courier to return the kit to us.
+
+When leaving the competition, teams should also be issued with [this form](https://github.com/srobo/kit-coordination-documents/tree/master/authorized-to-keep-kit), ensuring they've got all the information they need to return the kit back to us.
+
+[Example email prompting teams about kit return](https://github.com/srobo/team-emails/blob/master/SR2019/2019-05-15-kit-chase-round-1.md)
```

Comment:
Maybe expand this to clarify that sometimes we need to chase kit, both from teams who had loan extensions and also from those who didn't turn up to the competition?